bot/exts/utils/reminders.py

```diff
@@ -270,7 +270,7 @@ async def new_reminder(
             }
         )
 
-        mention_string = f"Your reminder will arrive {discord_timestamp(expiration, TimestampFormats.RELATIVE)}"
+        mention_string = f"Your reminder will arrive {discord_timestamp(expiration, TimestampFormats.DAY_TIME)}"
```

Comment:
I think `Your reminder will arrive on Thursday, January 1, 1970 1:00 AM` sounds better than `Your reminder will arrive Thursday, January 1, 1970 1:00 AM`

```suggestion
        mention_string = f"Your reminder will arrive on {discord_timestamp(expiration, TimestampFormats.DAY_TIME)}"
```

Reply:
Done :+1: